Source/ChocolateyGui.Common/Properties/Resources.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace ChocolateyGui.Common.Properties {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "17.0.0.0")]

```

Comment:
Had a chat with Gary about this. At this point, we shouldn't update the version of the Resource Builder being used.

Not until we only officially support building the project through VS 2022.

Reply:
That's what I figured. It added it automatically when I saved the resource file in Visual Studio. I've taken this commit out, but left the typo change in the comment as it's own commit.